web/src/components/Collapsible.js

```diff
@@ -23,11 +23,14 @@ class Collapsible extends Component {
       children,
       id,
       nested,
+      open: openProp,
       sticky,
       title,
       onChange
     } = this.props;
-    const isOpen = onChange ? this.props.open : this.state.open;
+    const { open: openState } = this.state;
```

Comment:
This is the main theme of the code changes.  Our eslint rules now require all React `state` and `prop` variables to be destructured - no more `this.props.blahblah`.

Reply:
Thanks for trudging through this and updating it!